pages/agent/v3/help/_annotate.md

```diff
@@ -30,7 +30,7 @@ Annotations are written in CommonMark-compliant Markdown, with &quot;GitHub
 Flavored Markdown&quot; extensions.
 
 The annotation body can be supplied as a command line argument, or by piping
-content into the command. The maximum size of each annotation body is 1MiB.
+content into the command. Custom Buildkite [emojis](https://buildkite.com/docs/pipelines/emojis) are supported. The maximum size of each annotation body is 1MiB.
```

Comment:
[here](https://github.com/buildkite/docs/blob/style-guide-updates-language-and-markdown-syntax/styleguide/markdown-syntax-style.md#links)

```suggestion
content into the command. Custom Buildkite [emojis](https://buildkite.com/docs/pipelines/emojis) are supported. The maximum size of each annotation body is 1MiB.
```

Reply:
Thanks @gilesgas! It looks like your suggested change has no difference, would you like me to make the change/push?

Reply:
🤦 I'm so sorry @mcncl , looks like I'd forgotten to amend my original 'Suggested change'. I did so just now and have committed it.

Reply:
The relativized version of the link worked (based on docs preview link).